htdocs/fourn/commande/orderstoinvoice.php

```diff
@@ -342,6 +347,14 @@
 	print '<tr><td>' . $langs->trans('PaymentMode') . '</td><td colspan="2">';
 	$html->select_types_paiements(isset($_POST['mode_reglement_id']) ? $_POST['mode_reglement_id'] : $mode_reglement_id, 'mode_reglement_id');
 	print '</td></tr>';
+  // Multicurrency
```

Comment:
Why do you have to ask the user the currency ?
The generated invoice is generated from order. If order was in currency X, invoice should be in currency X, if order is in currency Y, invoice should be in currency Y (invoice should match order), so why asking customer ?